float32[] goal_pose
float32 speed
bool wait
---
string message